api/src/controllers/transferCredits.ts

```diff
@@ -1,9 +1,37 @@
-import { router } from '../helpers/trpc';
+import { router, publicProcedure } from '../helpers/trpc';
+import { ANTEATER_API_REQUEST_HEADERS } from '../helpers/headers';
+
+interface APExam {
+  fullName: string;
+  catalogueName: string;
+  rewards: {
+    acceptableScores: number[];
+    unitsGranted: number;
+    electiveUnitsGranted: number;
+    geCategories: string[];
+    coursesGranted: {
+      string: string[];
+    };
+  }[];
+}
```

Comment:
When you `pnpm install`, API responses have automatically generated types. You should be able to do this to get the type instead of redefining the interface yourself:

```ts
import { operations } from "./generated/anteater-api-types";
export type APExam = operations['apExams']['responses'][200]['content']['application/json']['data']
```